src/commands/NFD.ts

```diff
@@ -111,23 +118,30 @@ class NFD {
 
     // Check for the cooldowns
     const ownerRecordPrev = await this.getUserFromDB(ownerMember.id)
-    if (ownerRecordPrev.lastMint.getTime() + this.MINT_COOLDOWN > Date.now()) {
-      return interaction.reply({
-        content: `Don't be greedy! You can hatch again <t:${Math.round(
-          (ownerRecordPrev.lastMint.getTime() + this.MINT_COOLDOWN) / 1000
-        )}:R>.`,
-        ephemeral: true,
-      })
-    }
-
-    const parts = await this.makeNFDcode()
-
-    // Check to see if we failed to make a unique one
-    if (!parts) {
-      return interaction.reply({
-        content: "I tried really hard but I wasn't able to make a unique dino for you. Sorry... :'(",
-        ephemeral: true,
-      })
+    if (this.MINT_COOLDOWN_METHOD == 'DAILY') {
```

Comment:
It's better to use `===` in javascript, avoids dumb conversions, even though it's not as bad in typescript because it will let you know.